server/index.js

```diff
@@ -1,8 +1,14 @@
 const express = require('express')
 const app = express()
+const db = require('../database_reviews/database.js')
 const port = 3001
 
 app.use(express.static('public'))
-app.get('/', (req, res) => res.send('Hello World!'))
+app.get('/api/:id/reviews', (req, res) => {
+    var allReview = function (data) {
+        res.send(data)
```

Comment:
Consider using `res.json` instead of `res.send`. Here, I don't think it make any difference functionally, but `res.json` appears to be more commonly used when sending back json data (it's a little more explicit). https://blog.fullstacktraining.com/res-json-vs-res-send-vs-res-end-in-express/